R/convert.R

```diff
@@ -16,19 +12,27 @@ col_ref <- list(
 #' @param frm A string, the input format of TCR data. Must be one of "tenx", "adaptive", "adaptivev2", or "imgt".
 #' @param to A string, the output format of TCR data. Must be one of "tenx", "adaptive", "adaptivev2", or "imgt".
 #' @param species A string, the species folder name under `tcrconvertr/inst/extdata/`. Optional; defaults to "human".
+#' @param verbose A boolean, whether to show messages.
 #'
 #' @return A string, the path to correct lookup table.
-choose_lookup <- function(frm, to, species = "human") {
+#' @export
+#' @keywords internal
+#' @examples
+#' choose_lookup("imgt", "adaptive")
+choose_lookup <- function(frm, to, species = "human", verbose) {
```

Comment:
I think `verbose` param should have a default value, yes? if not, then it should be placed to the left of the parameters WITH default values, with `frm` and `to`

Reply:
Yes, this was an oversight on my part. I fixed this and added tests in e968cf790898e678da8d58d9a37837b257bb09a1.